common/display/displayqueue.cpp

```diff
@@ -701,6 +701,13 @@ bool DisplayQueue::QueueUpdate(std::vector<HwcLayer*>& source_layers,
     idle_frame = false;
   }
 
+  // Force validate layers and skip GetCachedLayers if there is
+  // display rotation. There is potential issue calculating overlay
+  // layer's display_frame of composition plane in GetCachedLayers.
+  if (plane_transform_ != kIdentity) {
+    validate_layers = true;
```

Comment:
We should not use it. We should fix GetCachedLayers bugs.
I am also looking at CetCacheLayers recently, in some situation, the displayframe of the composition plane state is not correct.
We can postpone this PR until it root caused